pkg/gatherers/conditional/gathering_rule.schema.json

```diff
@@ -0,0 +1,127 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "gathering_rule.schema.json",
+    "title": "Conditional Gatherer Rule",
+    "descriptions": "A gathering rule for conditional gatherer",
+    "examples": [
+        {
+            "conditions": [
+                {
+                    "type": "alert_is_firing",
+                    "params": {
+                        "name": "SamplesImagestreamImportFailing"
+                    }
+                }
+            ],
+            "gathering_functions": {
+                "gather_logs_of_namespace": {
+                    "namespace": "openshift-cluster-samples-operator",
+                    "keep_lines": 100,
+                    "label_selector": ""
+                },
+                "gather_imagestreams_of_namespace": {
+                    "namespace": "openshift-cluster-samples-operator"
+                }
+            }
+        }
+    ],
+    "type": "object",
+    "required": [
+        "conditions",
+        "gathering_functions"
+    ],
+    "properties": {
+        "conditions": {
+            "type": "array",
+            "description": "The list of conditions on which the rule is triggered",
+            "minItems": 0,
+            "maxItems": 8,
```

Comment:
Does it make sense to limit it now?

Reply:
I think it's better to limit everything we can

Reply:
OK :)